src/gritql-token-provider.ts

```diff
@@ -0,0 +1,113 @@
+import * as vscode from "vscode";
+
+// TODO:  Use tree-sitter for more accurate parsing once tree-sitter-gritql exports wasm file
```

Comment:
This Todo should also explain how to address itself. Like, there's a reason why you couldn't implement it in this PR